### DISCUSSION TITLE
apisix dashboard

### DISCUSSION THREAD

**Discussion by w568590301** [apache/apisix#5866] [General]
apisix dashboard插件列表为何没有redirect插件？

**Comment by tokers** [comment]
@bzp2010 Could you check it out? Thanks!

**Reply by juzhiyuan** [reply]
Hi @w568590301, see

![image](https://user-images.githubusercontent.com/2106987/147076331-a9ab674b-2463-454f-828d-3f00513d0ca9.png)
